grader/util.py

```diff
@@ -1,7 +1,7 @@
 from . import cmd_completer
 from . import configfile
 
-IDENTITIES = (0, 1, 2)
+IDENTITIES = (0, 1, 2, 3)
```

Comment:
This was always supper annoying. I was thinking about changing to something like $USER, but never got around to it.
But this is good enough for now, we can do the pretty thing later.

Reply:
@keszybz I have two questions about this:

1) is this the only needed change for 4 identities to work? Or are we in for some nasty surprises?
2) just to double check, the final score is the average of the scores? For example, if some applications got 2 reviews, some 3, and some 4, will the scores remain comparable? We are asking because some of the applicants are known to some of the reviewers and so they want to exclude themselves from reviewing those applications. 

Bonus third question: is there a convenient way to say, "skip this application and also don't show this reviewer this application again"?

Thanks!

Reply:
> 1. s this the only needed change for 4 identities to work?

I expect so. I looked over the code, and it always is either a `for` loop over all identities, or `.mean()`, so the length of the vector shouldn't matter.

> 2. just to double check, the final score is the average of the scores? For example, if some applications got 2 reviews, some 3, and some 4, will the scores remain comparable?

So far the assumption was that we take the mean. When calculating, current code just takes the list of available gradings and takes the mean. If the list is empty, is returns nan. So this works fine even if some scores are unavailable.

But grader would bug you to always enter scores for all people. So I added an explicit "abstain" mechanism where the score is set to nan and `grade motivation` will skip those people.